### PR TITLE
Add .share to tensor in introduction.ipynb

### LINKED ISSUE #71: Add "share" showcase to the introduction tutorial
## Description
We can call ```share``` on a ```torch.tensor```. The method will return an ```MPCTensor```.
Showcase this usage in the introductory tutorial since it is more intuitive for the normal user.

## Are you interested in working on this improvement yourself?
- Yes, I am.

## Additional Context
The tutorial can be found [here](https://github.com/OpenMined/SyMPC/blob/main/examples/introduction.ipynb)

### PR DESCRIPTION
**Pull request by jmaunon**
## Description
Update `examples/introduction.ipynb` to call `.share` method directly from `Torch.tensor`.

Closes #71 

## Affected Dependencies
None

## How has this been tested?
Run the whole notebook without errors.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by gmuraru** [comment]
Ahhh -- could you also remove the ```Session``` and only pass the ```parties```?
Also, what do you think of having another cell where we showcase the ```MPCTensor``` and one cell with the ```share``` directly?

**Comment by jmaunon** [comment]
> Ahhh -- could you also remove the ```Session``` and only pass the ```parties```?
> Also, what do you think of having another cell where we showcase the ```MPCTensor``` and one cell with the ```share``` directly?

Sure! I will take a look later

**Comment by gmuraru** [comment]
Pushed the branch to be up-to-date with main.

**Comment by jmaunon** [comment]
@gmuraru I have included three ways of share a secret:

1. `tensor.share(clients)`
2. `tensor.share(session)`. 
3. `MPCTensor(secret, session)`

**Comment by gmuraru** [comment]
Merging this! I do not know why there are hanging tests on ```python-tests3.8```, but will fix it in the ```main``` branch if this is still happening.
